extensions/gmail/src/unreadmailsmenu.tsx

```diff
@@ -82,7 +86,7 @@ function UnreadMenuCommand(): JSX.Element {
         />
       </MenuBarExtra.Section>
       <MenuBarExtra.Section>
-        {!error && unreadCount <= 0 && <MenuBarExtra.Item title="No Unread Mails" />}
+        {!error && unreadCount <= 0 && <MenuBarExtra.Item title="No Unread Mails" icon={Icon.Envelope} />}
```

Comment:
Nitpick: We can use `Color.Red` or `Color.Orange` (with `Color` imported) for the icon in these cases to differentiate them.

```suggestion
        {!error && !unreadCount && <MenuBarExtra.Item title="No Unread Mails" icon={{ source: Icon.Envelope, tintColor: Color.Red }} />}
```

Reply:
No quite sure why I should make them red. They are already grayed out if there a no mail to read

Reply:
Oh yeah sorry, forgot that Menu Bar Extra already does that. Nvm this comment!